ITEM_en.tsv

```diff
@@ -173,13 +173,13 @@ ITEM_20150317_000187	Aston Two-Handed Sword
 ITEM_20150317_000188	Devi Two-Handed Sword
 ITEM_20150317_000189	Prima Two-Handed Sword
 ITEM_20150317_000190	Flamini
-ITEM_20150317_000191	A good sword that has long been regarded as essential item when facing Demons.
+ITEM_20150317_000191	A good sword that has long been regarded as an essential item when facing Demons.
 ITEM_20150317_000192	Wizard Slayer
 ITEM_20150317_000193	Novice Wizards often pay no mind to this particular weapon. However, a seasoned one knows to steer away from its unseen power.
 ITEM_20150317_000194	Two-Handed Sword [Strike]
-ITEM_20150317_000195	Like a flying dagger thrown in the darkest hours of the night, Naktis deals unforeseen blow to its target.
+ITEM_20150317_000195	Like a flying dagger thrown in the darkest hours of the night, Naktis deals unforeseen blows to its target.
 ITEM_20150317_000196	Luciduce
-ITEM_20150317_000197	Although some people dismiss this weapon calling it a 'tree cutting axe', those who actually used it would tell you that without it, the Great Plant Cataclysm would have resulted in far worse losses.
+ITEM_20150317_000197	Although some people dismiss this weapon as a 'tree cutting axe'; those who actually used it would tell you that without it, the Great Plant Cataclysm would have resulted in far worse losses.
```

Comment:
The semicolon is used completely improperly here. Semicolons separate two distinct sentences that are related, and the "Although" means that the first clause is not complete. The comma explains the "Although".